eth2util/signing/signing.go

```diff
@@ -229,3 +245,22 @@ func epochFromSlot(ctx context.Context, eth2Cl eth2wrap.Client, slot eth2p0.Slot
 
 	return eth2p0.Epoch(uint64(slot) / slotsPerEpoch), nil
 }
+
+// SlotHashRoot returns the ssz hash root of the slot.
+func SlotHashRoot(slot eth2p0.Slot) ([32]byte, error) {
```

Comment:
nit: I'd move this to `eth2util` since this isn't purely related to signing only.

Reply:
sure